### PR TITLE
Update FIRST libraries

### PR DESCRIPTION
**Pull request by buggy213**
Exactly as the title says.

### CODE REVIEW

**Comment by buggy213** [comment]
also, why is there a folder named skynet under your library? things are not quite as they seem lol.

**Comment by KevinHenriquez** [comment]
Is this a reply to me or the respiratory in rhsnutrons

**Comment by buggy213** [comment]
Um. This is a pull request for this repo to update the libaries to the newest official FIRST ones. I'm from team The Clueless

**Comment by buggy213** [comment]
Is this repository still active?

**Comment by KevinHenriquez** [comment]
16-17 is active also it's base code we took from first

**Comment by ehsanmasdar** [comment]
Hi @buggy213. Thanks for the PR. I'll get this looked at and merged in ASAP. CC @smo-key

**Comment by buggy213** [comment]
Cool. Our team looks forward to using this for our robot

**Comment by buggy213** [comment]
It appears there are several issues regarding the EncodedMotors.java class and the fact that the DcMotor class has been turned into an abstract class. I will look into this further tomorrow.

**Comment by buggy213** [comment]
Ok, the issues should be fixed now, although Travis CI is still spitting out errors because this repo is still using a old version of Gradle

**Comment by smo-key** [comment]
Hello @buggy213. Please try again by setting the Gradle version to `2.14.1` in `/home/travis/build/lasarobotics/FTCLibrary/ftc-library/build.gradle` and any other Gradle file in the repo.

Note: you may need Android Studio 2 or higher in order to use the newer version of Gradle. If you have trouble, let us know what version of Android Studio you used, then update to the latest version.

File: ftc-library/src/main/java/com/lasarobotics/library/skynet/EncodedMotor.java [hunk 1]
```diff
@@ -65,14 +66,14 @@ public boolean hasReachedPosition(double position) {
 
     public void enableEncoder() {
         //This command requires (at least ?) one execution loop to reset the encoders
-        super.setMode(DcMotorController.RunMode.RUN_USING_ENCODERS);
+        super.setMode(RunMode.RUN_WITHOUT_ENCODER);
```
Comment (smo-key) [inline ftc-library/src/main/java/com/lasarobotics/library/skynet/EncodedMotor.java:69]: @buggy213 Is this supposed to be `RUN_WITH_ENCODER`?

Comment (buggy213) [reply ftc-library/src/main/java/com/lasarobotics/library/skynet/EncodedMotor.java:69]: yeah, good catch

**Comment by ehsanmasdar** [comment]
Fixed in #50

**Pull request closed without merging**